doc/modules/sortingcomponents.rst

```diff
@@ -136,6 +152,85 @@ For instance, the 'monopolar_triangulation' method will have:
      * **'z'** is orthogonal to the probe plane
 
 
+Node pipelines
+--------------
+
+Both :py:func:`~spikeinterface.sortingcomponents.peak_detection.detect_peaks()` and
+:py:func:`~spikeinterface.sortingcomponents.peak_localization.localize_peaks()` need to walk throughout the entire
+recording traces. So this make reading traces and computing the preprocessing twice : this can be very slow!
+Luckily, there is an internal machinery to avoid the 2 times traces reading : :py:func:`~spikeinterface.core.run_node_pipeline()`
```

Comment:
```suggestion
recording traces, duplicating the reading of the traces from disk and applying the preprocessing.
This can be very slow!

Luckily, there is an internal machinery to avoid the multiple passes over the traces: the :py:func:`~spikeinterface.core.run_node_pipeline()` function.
```